### PR TITLE
Add inside rust blog about WG-learning

### PR DESCRIPTION
**Pull request by mark-i-m**

### CODE REVIEW

**Comment by mark-i-m** [comment]
r? @spastorino 

cc @rust-lang/wg-learning

File: posts/inside-rust/2019-12-18-wg-learning-update.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+guide has some long-standing holes and shortcomings that we would like to address.
+
+Specifically, the Learning WG decided that we wanted to pursue the following goals next:
+- Write an overview chapter
+- Gather source material for chapters on monomorphization and LLVM
+
+### Overview chapter
+
+One of the challenges with big software systems is understanding how everything
+fits together. We seen this problem come up with the rustc-guide; the chapters
```
Comment (JohnTitor) [inline posts/inside-rust/2019-12-18-wg-learning-update.md]:
```suggestion
fits together. We have seen this problem come up with the rustc-guide; the chapters
```

`have seen` is more appropriate?

**Comment by spastorino** [comment]
This looks great, pinging Niko to merge ...

r? @nikomatsakis

**Comment by spastorino** [comment]
@nikomatsakis yes

**Pull request merged**